gcalcli

```diff
@@ -1377,12 +1388,22 @@ class gcalcli:
                     newStart, newEnd = GetTimeFromStr(val.strip(), length)
                     event['s'] = parse(newStart)
                     event['e'] = parse(newEnd)
-                    event['start'] = \
-                        {'dateTime': newStart,
-                         'timeZone': event['gcalcli_cal']['timeZone']}
-                    event['end'] = \
-                        {'dateTime': newEnd,
-                         'timeZone': event['gcalcli_cal']['timeZone']}
+
+                    if FLAGS.allday:
```

Comment:
I'd remove the various `'dateTime': None`, `'timeZone': None`, & `'date': None` declarations as they should be redundant.  If they don't exist in the dictionary, their value is `None`.